imports/ui/components/chat-box.js

```diff
@@ -108,9 +109,9 @@ class ChatBox extends React.Component {
 }
 
 ChatBox.propTypes = {
-  chats: React.PropTypes.array,
-  routeParams: React.PropTypes.string,
-  wireid: React.PropTypes.string,
+  chats: PropTypes.array,
+  routeParams: PropTypes.string,
+  wireid: PropTypes.string,
```

Comment:
Why not use camelcase here as well? E.g: wireId

Reply:
Check the prop-types package. It's the same reason we write React with capital R. https://www.npmjs.com/package/prop-types

Reply:
I misunderstood. you meant using `wireId` etc. I'll take a look about that. That's a different issue than PropTypes.